codesign/bin/codesign.dart

```diff
@@ -30,17 +28,16 @@ const String kGCloudUploadPattern = 'google-cloud-upload-pattern';
 /// Otherwise, nothing will be uploaded back for dry run. default value is
 /// true.
 ///
-/// For [kCodesignFilepathOption], supply the filepaths of binaries to be codesigned.
-/// e.g. supply `--filepath=darwin-x86/artifacts.zip --ios-release/artifacts.zip` if you would like to codesign darwin-x86/artifacts.zip and ios-release/artifacts.zip
-///
-/// For [kGCloudDownloadPattern], it is a required parameter to specify the google cloud bucket prefix of the artifacts stored in google cloud.
-/// Artifacts stored with this bucket prefix path are downloaded from google cloud and codesigned.
-/// For [kGCloudUploadPattern], it is a required parameter to specify the google cloud bucket prefix for the code signed artifacts to be uploaded to.
-/// Code signed artifacts are uploaded to google cloud under this bucket prefix.
+/// For [kGCloudDownloadUploadPathOption], it is a required parameter to specify the google cloud bucket paths.
```

Comment:
This should be two separate options (a download path and an upload path)

Reply:
Sure, done.

Reply:
(sneak in one extra comment: Casey looks very strong in the new github profile picture)